src/pages/UpdateServers.js

```diff
@@ -462,21 +497,53 @@ class UpdateServers extends BaseUpdateWizardPage {
     }
   }
 
-  deleteComputeHost(id) {
+  async deleteComputeHost(id) {
+    let server = this.state.serverStatuses[id];
     this.setState({
       confirmDelete: {
         show: true,
-        loading: false,
-        id: id
+        loading: true,
+        id: id,
+        osPassword: ''
       }
     });
+    try {
+      let promises = [getReachability(server['ip-addr'])];
+      let [conectivityStatus] = await Promise.all(promises);
+      this.setState(prev => ({
+        serverStatuses: {
```

Comment:
its been a while since I mucked around with setting state, is it really necessary to explicitly set the statuses, ID, and the internal states just to set the isReachable state in the internal object? seems like there might be a more straightforward way to do this...

Reply:
I just did a quick fix for the problem in delete compute.  I picked up the code from deactivate so that at least they are in consistent code style. I am sure it could be made to be straight forward...however this seems to work at this point.